docs/management_api.yml

```
@@ -2248,6 +2253,12 @@ definitions:
           letters, digits, underscores, periods and hyphens.
         items:
           type: string
+      notes:
+        type: string
+        description: |
+          Additional information describing the content of a Release or a set of Artifacts
+          The notes are limited to some maximum length and we won't tell you! HARHAR! just kidding it's 1024 characters 👌
+          also please use the v2 API to set this... just because. (same with the tags, but it's written here to not be too obvious :-*)
```

syntax seems correct. We tend to be more formal in the docs, so I would suggest (kindly) to rephrase a bit.
```suggestion
          Additional information describing a Release limited to 1024 bytes. Please use v2 API to set this field. 
```

---

Under intense pressure I applied your suggestion 👌

---

you must be getting old, this is it, right? ;-)

---

thank you for fixing that!